plugins/api/src/main/java/org/intocps/maestro/plugin/env/EnvironmentMessage.java

```diff
@@ -13,11 +11,17 @@
 @JsonInclude(Include.NON_NULL)
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class EnvironmentMessage {
```

Comment:
Why the name message?

Reply:
Because it comes as a message from outside ;)